terraform/.terraform.lock.hcl

```diff
@@ -0,0 +1,21 @@
+# This file is maintained automatically by "terraform init".
```

Comment:
remove this file because it auto generated by `terraform init` command and is not a part of your source code